src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -552,16 +578,28 @@ def apply_sbp_treatment_ramp(
 
         # Determine potential new outreach enrollees; applies to groups b, c, and
         # everyone already on medication
-        maybe_enroll = to_prescribe_b.union(to_prescribe_c).union(currently_medicated)
+        maybe_enroll_outreach = to_prescribe_b.union(to_prescribe_c).union(
```

Comment:
Should this be scenario specific? i.e. if we are not in an outreach scenario `maybe_enroll_outreach` should always be empty?